flow-report/test/setup/global-setup.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export default async () => {
+  process.env.TZ = 'UTC';
```

Comment:
maybe add a comment on the reason for this? Took me a little while to figure out the purpose when there's no other reference to `process.env.TZ` :)